docs/docs/.vuepress/configs/sidebarConfig.js

```diff
@@ -8,6 +8,8 @@ const docsNav = [
       "/workflows/",
       "/workflows/steps/",
       "/workflows/steps/triggers/",
+      "/workflows/steps/actions/",
```

Comment:
Trying to hit this page directly yields the main docs page. Is that related to the issue you were troubleshooting last week?

https://pipedream-docs-5faq8yv4n-pipedreamers.vercel.app/docs/workflows/steps/actions/

Reply:
Fixed!